### DISCUSSION TITLE
doc

### DISCUSSION THREAD

**Discussion by AdwardAllan** [Nek5000/nekRS#374] [General]
if there have documents which similiar to netdoc for nek5000,I can't build the run case and boundary conditions

**Comment by stgeke** [comment]
Currently there is no userguide available but we're working on it.

**Comment by zongzilin** [comment]
nek5000 doc can give you some help

**Comment by AdwardAllan** [comment]
Thanks,  Now  I have run eaxmples with NekRS,  But   I want to know how to Make GPU available.
